src/locales/en.json

```diff
@@ -70,7 +70,7 @@
   "contactToPlace": {
     "saveStartPlace": "Save the starting point?",
     "saveEndPlace": "Save the arrival point?",
-    "tooltip": "To automate this categorisation in the future, save this address in a contact record",
+    "tooltip": "To make this categorisation more reliable in the future, record this address in a contact record.",
```

Comment:
"record [...] in a contact record" sounds weird. I suggest "register this address in a contact record"